scripts/exportFromSketch.js

```diff
@@ -88,12 +76,37 @@ exec(
 
 		const filesToExport = diffToArray(result);
 
-		if (filesToExport.length) {
-			console.info(`Exporting ${filesToExport} as ${FORMAT} for ${PLATFORM}`);
-			exportFiles(filesToExport);
-		} else {
+		if (!filesToExport.length) {
 			console.info('\nNo sketch changes found, skipping build\n');
```

Comment:
Let's write a clearer message. The build still runs, it just doesn't actually export anything

Reply:
going to remove this message completely, as the next PR removes the git diff checking and we just run the export on all sketch files every time